tests/test_source_file/test_source_file.py

```diff
@@ -1,119 +1,97 @@
 #!/usr/bin/env python
 
-import os
 import sys
-from subprocess import Popen, STDOUT, PIPE, call
-import filecmp
-import glob
-from optparse import OptionParser
-
-parser = OptionParser()
-parser.add_option('--mpi_exec', dest='mpi_exec', default='')
-parser.add_option('--mpi_np', dest='mpi_np', default='3')
-parser.add_option('--exe', dest='exe')
-(opts, args) = parser.parse_args()
-cwd = os.getcwd()
+sys.path.insert(0, '..')
+from testing_harness import *
+
 
 settings1="""<?xml version="1.0"?>
 <settings>
-
   <state_point batches="10" />
   <source_point separate="true" />
-
   <eigenvalue>
     <batches>10</batches>
     <inactive>5</inactive>
     <particles>1000</particles>
   </eigenvalue>
-
   <source>
     <space type="box">
       <parameters>-4 -4 -4  4  4  4</parameters>
     </space>
   </source>
-
 </settings>
 """
 
 settings2 = """<?xml version="1.0"?>
```

Comment:
We need to run OpenMC with two different inputs.  The first creates a source_point file.  The second loads that source_point for the initial source.